### DISCUSSION TITLE
Nightly releases in main repo

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1354] [Q&A] [answered]
Does Github allow to publish nightly releases to as a different tag? like to have normal public and then a ton of nightly in the same repo

**Comment by kagikn** [comment, marked as answer]
Possible, but I created a nightly release repo because you can't exclude prereleases from the release notification. If GitHub added the option to drop notifications for prereleases, maybe I consider pushing nightly tags to the main repo.

**Reply by Michael21107** [reply]
gotcha, thanks
